Jobs/src/Job.IdentityDeletion/Workers/ActualDeletionWorker.cs

```diff
@@ -65,6 +72,7 @@ private async Task ExecuteDeletion(CancellationToken cancellationToken, Identity
     {
         await NotifyIdentityAboutStartingDeletion(cancellationToken, identityAddress);
         await Delete(identityAddress);
+        await _deviceIdentityDeleter!.Delete(identityAddress, _deletionProcessLogger);
```

Comment:
This could maybe benefit from extraction into a separate method whose name would clarify the intention behind this.

Reply:
Yea that makes sense. Changes made.